mne_bids/mne_bids.py

```diff
@@ -594,18 +565,6 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
     data_meta_fname = make_bids_filename(
```

Comment:
`data_meta_fname` should be `sidecar_fname` in my opinion

Reply:
+1
I feel this should have been renamed a while ago...

Reply:
Let's do this elsewhere. It's not the purpose of this pull request ...

Reply:
Actually I addressed this so we can focus on more important design issues :)